menu.py

```diff
@@ -47,6 +49,35 @@ def menu():
             case "5":
                 add_new_product()
 
+            case "6":
+                sort_option = input("""
+            Wybierz sposób sortowania:
+            [a] Cena rosnąco
+            [b] Cena malejąco
+            [c] Producent A-Z
+            [d] Producent Z-A
+            [e] Nazwa A-Z
+            [f] Nazwa Z-A
+            [g] Czas dostawy (malejąco)
```

Comment:
"Czas dostawy (malejąco)"  does not output a thing, the variable of delivery is not yet(?) set so I do not think that user should have that particular option.

Reply:
Sure, I'm deleting it. I thought that I'd leave it and after the "czas dostawy" is further developped input it, but you're right. There's no need for it to be in the menu as for now.